.goreleaser.yml

```diff
@@ -0,0 +1,58 @@
+before:
+  hooks:
+    - go mod download
+builds:
+- id: botkube
+  binary: botkube
+  main: cmd/botkube/main.go
+  ldflags: &ldflags
+  - -s -w
+    -X github.com/infracloudio/botkube/pkg/version.Version={{ .Tag }}
+    -X github.com/infracloudio/botkube/pkg/version.GitCommitID={{ .Commit }}
+    -X github.com/infracloudio/botkube/pkg/version.BuildDate={{ .Date }}
+  env:
+  - CGO_ENABLED=0
+  goos:
+    - linux
+  goarch:
+    - amd64
+    - arm
+    - arm64
+  goarm:
+    - 7
+snapshot:
+  name_template: "{{ .Tag }}"
+changelog:
+  skip: true
+dockers:
+- image_templates:
+  - 'ghcr.io/infracloudio/botkube:{{ .Tag }}-amd64'
+  use: buildx
+  dockerfile: 'build/Dockerfile'
+  build_flag_templates:
+  - "--platform=linux/amd64"
+  - "--build-arg=botkube_version={{ .Tag }}"
+- image_templates:
+  - 'ghcr.io/infracloudio/botkube:{{ .Tag }}-arm64'
+  use: buildx
+  goarch: arm64
+  dockerfile: 'build/Dockerfile'
+  build_flag_templates:
+  - "--platform=linux/arm64"
+  - "--build-arg=botkube_version={{ .Tag }}"
+- image_templates:
+  - 'ghcr.io/infracloudio/botkube:{{ .Tag }}-armv7'
+  use: buildx
+  goarch: arm
+  goarm: 7
+  dockerfile: 'build/Dockerfile'
+  build_flag_templates:
+  - "--platform=linux/arm"
+  - "--build-arg=botkube_version={{ .Tag }}"
+
+docker_manifests:
+- name_template: ghcr.io/infracloudio/botkube:{{ .Tag }}
```

Comment:
Final image which we are going to push would be following right ?
```docker tag infracloudio/botkube:latest-amd64 <your_account>/botkube:latest```

It wont have tags like `{{ .Tag }}-amd64` etc ?

Reply:
@sanketsudake goreleaser will push all the images - `{{ .Tag }}-amd64`, `{{ .Tag }}-arm64` and `{{ .Tag }}-armv7`. And finally create a manifest with tag `{{ .Tag }}`. 
https://goreleaser.com/customization/docker_manifest/